### ISSUE TITLE
[Feedback] Wrist overlay counting yourself as a friend may lead to unnecessary confusion

### ISSUE THREAD

**Issue by SalbugVR** [Natsumi-sama/VRCX#8]
Basically what the title says . I feel having the wrist counter add a tick for you to the friend counter may be unnecessary especially if you're trying to get a quick count of how many of your friends are in the instance and (now with this recent update) making sure to always count one down to not include yourself in the count.

**Comment by Natsumi-sama** [comment]
Hmmm the only reason I did this change was to make the number consistent with the instance counter on the user/world dialog.

**Comment by SalbugVR** [comment]
> Hmmm the only reason I did this change was to make the number consistent with the instance counter on the user/world dialog.

I'm noticing that now actually. Is it necessary to even include yourself as a friend in the user/world dialog counter as well though or is that just an API thing? The reason I bring this up is I just feel it's kind of unnecessary bloat to the in-instance friend counter.

**Comment by Natsumi-sama** [comment]
Yup that was the reason I chose to add yourself to wrist overlay instead of removing from dialog instance counter because it will require a lot more code to do, I might do it though if it's better to not include yourself it will just be a pain.

**Comment by SalbugVR** [comment]
> Yup that was the reason I chose to add yourself to wrist overlay instead of removing from dialog instance counter because it will require a lot more code to do, I might do it though if it's better to not include yourself it will just be a pain.

I figured that would've been the case. If you decide to do it though, your efforts definitely won't go unappreciated. ^^

**Comment by Natsumi-sama** [comment]
https://github.com/Natsumi-sama/VRCX/commit/3b07a54bc67684c9030fe5569df1c010a7bd3613 yup that was annoying to do, and may give broken friend count when using an alt account depending if they are friends with main account or not but eh good enough.
